glide.yaml

```diff
@@ -21,7 +21,7 @@ import:
   - json
 - package: github.com/projectcalico/go-yaml-wrapper
 - package: github.com/projectcalico/libcalico-go
-  version: 1.2.0
+  version: 1.2.1-rc1
```

Comment:
this should be `v1.2.1-rc1`, missing the `v` (glide might be smart enough to figure it out I guess, but we should make is explicit)